src/Avalonia.Controls/ProgressBar.cs

```diff
@@ -38,6 +41,7 @@ static ProgressBar()
             PseudoClass<ProgressBar, Orientation>(OrientationProperty, o => o == Orientation.Vertical, ":vertical");
             PseudoClass<ProgressBar, Orientation>(OrientationProperty, o => o == Orientation.Horizontal, ":horizontal");
             PseudoClass<ProgressBar>(IsIndeterminateProperty, ":indeterminate");
+            PseudoClass<ProgressBar>(ShowProgressTextProperty, ":showprogresstext");
```

Comment:
This pseudoclass doesn't seemed to be used. Is it needed?

Reply:
Maybe in the future? I've added for safety and because all others have pseudoclasses but I can also remove it?!

Reply:
Yeah, I'd remove it if it's not used. Things can be added in future more easily than they can be removed. YAGNI and all that.

Reply:
Also the `PseudoClass` method has been removed because it caused performance issues ;)